bin/run.sh

```diff
@@ -1,40 +1,29 @@
-#! /bin/sh
-set -e
+#!/usr/bin/env bash
 
-# If arguments not provided, print usage and exit
+# Synopsis:
+# Run the test runner on a solution.
+
+# Arguments:
+# $1: exercise slug
+# $2: absolute path to solution folder
+# $3: absolute path to output directory
```

Comment:
I've changed this file to take positional arguments as that is how the interface requires it.